app/controllers/documents_controller.rb

```diff
@@ -39,6 +39,30 @@ def create
       else
         render action: 'new_folder'
       end
+    elsif params[:document][:file].kind_of?(Array)
```

Comment:
Hey @cessien this controller will definitely need to be refactored. I will give a hand when I am free perhaps?